name.py

```diff
@@ -0,0 +1,6 @@
+name = input("escribe tu nombre bro :")
+last_name + inputt("escribe tu apellido bro:")
```

Comment:
oye hector quieres salir conmigo ??
```suggestion
last_name = input("escribe tu apellido bro:")
```